src/addresses.ts

```diff
@@ -0,0 +1,244 @@
+import { ChainId, SUPPORTED_CHAINS, SupportedChainsType } from './chains'
+
+type AddressMap = { [chainId: number]: string }
+
+type ChainAddresses = {
+  v3CoreFactoryAddress: string
+  multicallAddress: string
+  quoterAddress: string
+  v3MigratorAddress?: string
+  nonfungiblePositionManagerAddress?: string
+  tickLensAddress?: string
+  swapRouter02Address?: string
+  v1MixedRouteQuoterAddress?: string
+}
+
+const DEFAULT_NETWORKS = [ChainId.MAINNET, ChainId.GOERLI, ChainId.SEPOLIA]
```

Comment:
what makes a network default? slightly confused that sepolia is part of this when it doesn't use the DEFAULT_ADDRESSES unlike goerli and mainnet

Reply:
good callout, it's mainly used in the `constructSameAddressMap` which I put Sepolia in as it shares a uni address, but this was a mistake as it doesn't share ens registrar for example. I will remove it from `DEFAULT_NETWORKS` and instead append it to the `UNI_ADDRESSES` map